# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../knetconf/gcc-4.1.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Not a new checkout, because the upstream netconf exact-build migration is not yet finished.

--- ./gui/ppp_device.hh	(revision 285)
+++ ./gui/ppp_device.hh	(revision 286)
@@ -32,7 +32,7 @@
   void selectionChanged (int item); // by ComboBox
 
 private:
-  void PPPDevice::SetPPPOE (bool value); // by refreshContent()
+  void SetPPPOE (bool value); // by refreshContent()
   void SwitchType ();  // common stuff by selectionChanged / SetPPPOE
 
   QString title;
